addons/sale_coupon/models/sale_order.py

```diff
@@ -510,7 +511,14 @@ def _compute_tax_id(self):
             taxes = line.tax_id.filtered(lambda r: not line.company_id or r.company_id == line.company_id)
             line.tax_id = fpos.map_tax(taxes, line.product_id, line.order_id.partner_shipping_id)
 
-    # Invalidation of `coupon.program.order_count`
+    def _get_display_price(self, product):
+        # A product created from a promotion does not have a list_price.
+        # The price_unit of a reward order line is computed by the promotion, so it can be used directly
+        if self.is_reward_line:
+            return self.price_unit
+        return super()._get_display_price(product)
+
+    # Invalidation of `sale.coupon.program.order_count`
```

Comment:
why adding `sale` here? It adds useless diff

Reply:
Oops, the entire line got added in my git differences, and as I saw that it was present in 13.0, I let that. I didn't see it was changed in 14.0